vm/src/cairo_run.rs

```diff
@@ -26,6 +29,7 @@ pub struct CairoRunConfig<'a> {
     pub trace_enabled: bool,
     pub relocate_mem: bool,
     pub layout: LayoutName,
+    pub cairo_layout_params: Option<CairoLayoutParams>,
```

Comment:
same here, I think `dynamic_layout_params` makes more clear to the user that is only needed in that cases
Also add a little comment here explaining like you did in the CairoRunner

Reply:
Done